fern/docs.yml

```diff
@@ -6,64 +6,193 @@ instances:
         owner: fern-api
         repo: fern
         branch: main
-  # - url: fern-beta.docs.buildwithfern.com/learn
-  #   custom-domain: staging.buildwithfern.com/learn
 title: Fern | Documentation
 
 tabs:
-  overview:
-    display-name: Overview
-    icon: "globe"
+  home:
+    display-name: Home
+    icon: globe
   sdks:
-    display-name: SDKs
+    display-name: Generate SDKs
     slug: sdks
     icon: code
   docs:
-    display-name: Docs
+    display-name: Generate docs
     slug: docs
     icon: book
+  config:
+    display-name: Configure
+    slug: config
+    icon: puzzle
+  api:
+    display-name: API reference
+    slug: api
+    icon: asterisk
 
 navigation:
-  - tab: overview
+  - tab: home
     layout:
       - page: Introduction
-        path: ./docs/pages/overview/introduction.mdx
-      - page: Why Fern?
-        path: ./docs/pages/overview/why-fern.mdx
-      - page: Get started
-        path: ./docs/pages/overview/get-started.mdx
-      - section: Define your API
+        path: ./overview/intro.mdx
+  - tab: sdks
+    layout:
+      - page: Overview
+        path: ./overview/dummy.mdx
+      - page: Quickstart
+        path: ./overview/dummy.mdx
+      - section: Features
+        contents:
+          - page: Strongly typed
+            path: ./overview/dummy.mdx
+          - page: Idiomatic method names
+            path: ./overview/dummy.mdx
+          - page: Schema Validation
+            path: ./overview/dummy.mdx
+          - page: Auto-Pagination
+            path: ./overview/dummy.mdx
+          - page: Retries with backoff
+            path: ./overview/dummy.mdx
+          - page: Webhook Signature Verification
+            path: ./overview/dummy.mdx
+          - page: Forward Compatibility
+            path: ./overview/dummy.mdx
+          - page: Multipart Form Data
+            path: ./overview/dummy.mdx
+          - page: Idempotency Headers
+            path: ./overview/dummy.mdx
+          - page: Server-Sent Events
+            path: ./overview/dummy.mdx
+          - page: Integration Tests
+            path: ./overview/dummy.mdx
+          - page: Code Snippets
+            path: ./overview/dummy.mdx
+          - page: Augment with custom code
+            path: ./overview/dummy.mdx
+          - page: Merging multiple APIs
+            path: ./overview/dummy.mdx
+          - page: GitHub integration
+            path: ./overview/dummy.mdx
+  - tab: docs
+    layout:
+      - page: Overview
+        path: ./overview/dummy.mdx
+      - page: Quickstart
+        path: ./overview/dummy.mdx
+      - section: Global Settings
```

Comment:
How about adding a page where we talk about SSO and password protection?

page: Site Protection
path: ./overview/protection.mdx